src/main/java/ch/njol/skript/effects/EffTeleport.java

```diff
@@ -38,31 +38,39 @@
 import ch.njol.util.Kleenean;
 import io.papermc.lib.PaperLib;
 import io.papermc.lib.environments.PaperEnvironment;
+import org.bukkit.Bukkit;
+import org.bukkit.Chunk;
 import org.bukkit.Location;
 import org.bukkit.entity.Entity;
 import org.bukkit.event.Event;
 import org.bukkit.event.player.PlayerMoveEvent;
 import org.bukkit.event.player.PlayerRespawnEvent;
 import org.eclipse.jdt.annotation.Nullable;
 
+import java.util.concurrent.CompletableFuture;
+
 @Name("Teleport")
 @Description({
 	"Teleport an entity to a specific location. ",
 	"This effect is delayed by default on Paper, meaning certain syntax such as the return effect for functions cannot be used after this effect.",
 	"The keyword 'force' indicates this effect will not be delayed, ",
-	"which may cause lag spikes or server crashes when using this effect to teleport entities to unloaded chunks."
-})
-@Examples({
-	"teleport the player to {homes.%player%}",
+	"which may cause lag spikes or server crashes when using this effect to teleport entities to unloaded chunks.",
+	"If your code doesn't need to wait for the teleport to finish before continuing, you can use the 'eventually' keyword",
+	"which indicates that your code should continue executing even if the teleport hasn't happened yet."})
+@Examples({"teleport the player to {homes::%player's uuid%}",
 	"teleport the attacker to the victim"
 })
-@Since("1.0")
+@Since("1.0, (INSERT VERSION for teleport eventually)")
 public class EffTeleport extends Effect {
 
 	private static final boolean CAN_RUN_ASYNC = PaperLib.getEnvironment() instanceof PaperEnvironment;
 
 	static {
-		Skript.registerEffect(EffTeleport.class, "[(1¦force)] teleport %entities% (to|%direction%) %location%");
+		Skript.registerEffect(EffTeleport.class,
+			"teleport %entities% (to|%direction%) %location%",
+					"(force|immediately) teleport %entities% (to|direction) %location%",
+					"teleport %entities% (to|direction) %location% (eventually|as soon as possible)"
+			);
 	}
```

Comment:
Let me know if I'm wrong but it looks like we're missing `%` on `direction` in the last 2 patterns, now it's just `teleport %entity% direction %location%`
```suggestion
		Skript.registerEffect(EffTeleport.class,
			"teleport %entities% (to|%direction%) %location%",
					"(force|immediately) teleport %entities% (to|%direction%) %location%",
					"teleport %entities% (to|%direction%) %location% (eventually|as soon as possible)"
			);
	}
```